joomla/joomla-cms
# [4.3] Fix issue Deprecated: trim(): Passing null on PHP 8.1

Pull Request for Issue #38510 .

### Summary of Changes
If $path is null, it will get '' value instead of run trim() function.


### Testing Instructions

1. Open templates/cassiopeia/index.php
2. Add code below at the line 17:
```
use Joomla\CMS\Filesystem\Path;
echo Path::clean(null);
```
3. Switch template to cassiopeia, go to front-end and check it.

### Actual result BEFORE applying this Pull Request
`Deprecated: trim(): Passing null to parameter #1 ($string) of type string is deprecated in /{root}/libraries/src/Filesystem/Path.php on line 198`

### Expected result AFTER applying this Pull Request
Warning disappear


### Documentation Changes Required
No
